tests/config/controller-definitions-config.yaml

```diff
@@ -1,10 +1,769 @@
 controllerDefinitions:
-  - version: "1.2.1"
+  - version: "1.2.0"
```

Comment:
Using this file in unit test, so that I can have at least two version of numaflow for testing